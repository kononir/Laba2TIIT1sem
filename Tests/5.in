23
32
1 17
1 11
11 17
17 5
1 5
11 5
5 3
20 3
20 21
21 3
22 3
3 10
22 10
2 22
3 7
7 13
13 6
7 16
16 6
6 8
8 19
8 23
23 19
19 18
18 14
14 19
14 23
23 15
23 9
9 4
4 12
12 9
