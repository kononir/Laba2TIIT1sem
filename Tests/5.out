6
7
14
15
20
28
29
8
3
5
6
7
8
9
22
23
